resources/views/auth/verify.blade.php

```diff
@@ -22,7 +22,11 @@
               @endif
 
               {{ trans('auth.confirmation_check') }}<br/>
-              {!! trans('auth.confirmation_request_another', ['url' => route('verification.resend')]) !!}<br/>
+              {!! trans('auth.confirmation_request_another') !!}<br/>
+              <form action="{{ route('verification.resend') }}" method="post" class="di">
```

Comment:
This is the fix here. It needs to call a form that triggers a POST request, therefore the use of the <form> as well as the submit button.